insights/specs/__init__.py

```diff
@@ -2,6 +2,8 @@
 
 
 class Specs(SpecSet):
+    # Client App specs
+    malware_detection = RegistryPoint()
```

Comment:
Could you add a blank line after this to separate it from the others. Otherwise everything else looks good to me!

Reply:
Done. Thanks for reviewing.